.github/workflows/render-md-mermaid.yml

```diff
@@ -0,0 +1,16 @@
+# https://github.com/nielsvaneck/render-md-mermaid#github-action--workflow-example
+name: render-md-mermaid
```

Comment:
will this action be executed when pushing it on forked repos? The publish-next.yml prevents running this action when it's a fork.

Reply:
Good point, so this action would have needed that guard too. Thanks for the flag. Now I'm about to remove this whole action file anyway, will push that next.